src/index.css

```diff
@@ -0,0 +1,387 @@
+*,
```

Comment:
Only SCSS file should be committed. `.css` and `.css.map` should be ignored with proper entry in `.gitignore`